docs/install.md

```diff
@@ -130,7 +117,6 @@ We publish self-contained .zip and .tar.gz archives in [GitHub releases](https:/
    coder server --postgres-url <url> --access-url <url>
    ```
 
-## Next steps
+## Up Next
 
-Once you've installed and started Coder, see the [quickstart](./quickstart.md)
-for instructions on creating your first template and workspace.
+Learn how to [configure Coder](./configure.md)
```

Comment:
```suggestion
Learn how to [configure Coder](./install/configure.md)
```

Are you on the `yarn dev` workflow in coder.com?

Reply:
@ammario I'm not - I'll work on learning how to use it.